README.md

```diff
@@ -1,2 +1,79 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Collaborative Project: The Next Web
+
+> This is a microverse student project for working with hierarchy of elements and responsive design
+
+![screenshot](https://github.com/Gnwin/Godwin-Eraani-Next-Web-project/blob/nextweb/assets/images/Screenshotnewtnw.png)
+
+> This project clones the thenextweb.com article page interface design.
+> The project includes the header, the main content, footer, media queries.
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo
+
+[Live Demo Link](https://raw.githack.com/Gnwin/Godwin-Eraani-Next-Web-project/nextweb/index.html)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+> Browser
+> Text editor of your choice.
+> npm has to be installed in your system.
+> install linters on your IDE.
+
+### Setup
+
+> `git clone https://github.com/Gnwin/Godwin-Eraani-Next-Web-project.git`
+
+### Usage
+
+> Check the nextweb branch.
+> The project was finished up on the nextweb branch
+> My partner contributed hers on nextbranch branch.
```

Comment:
Since the readme file will be the official document on the default branch of your project, there is no need to talk about your `nextweb branch`, it will not stay there for ever, after your PR is merged to the main branch, then there will be no need to checkout that `nextweb branch`. So, I highly recommend you to remove this `nextweb branch` related information from your readme file